src/main.cpp

```diff
@@ -3628,6 +3638,12 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
     else if (strCommand == "verack")
     {
         pfrom->SetRecvVersion(min(pfrom->nVersion, PROTOCOL_VERSION));
+
+        // Update the last seen time for this node's address
```

Comment:
This comment is no longer correct, should probably be `Update the connected status for this node`